STL_Extension/include/CGAL/STL_Extension/internal/parameters_interface.h

```diff
@@ -220,6 +220,8 @@ CGAL_add_named_parameter(scan_angle_t, scan_angle_map, scan_angle_map)
 CGAL_add_named_parameter(scanline_id_t, scanline_id_map, scanline_id_map)
 CGAL_add_named_parameter(min_points_per_cell_t, min_points_per_cell, min_points_per_cell)
 CGAL_add_named_parameter(scalar_t, scalar_map, scalar_map)
+CGAL_add_named_parameter(apply_iterative_snap_rounding_t, apply_iterative_snap_rounding, apply_iterative_snap_rounding)
```

Comment:
The new parameters are not in the right block. They should go in the block starting with the comment
```
// List of named parameters used in the package 'Polygon Mesh Processing'
```